giraffe/src/utils/newTable.ts

```diff
@@ -2,7 +2,7 @@ import {Table, ColumnType, ColumnData, Config, FluxDataType} from '../types'
 import {fromFlux} from './fromFlux'
 
 // Don't export me!
-class SimpleTable implements Table {
+class EmpiricalTable implements Table {
```

Comment:
This was very confusing. An internal `SimpleTable` class that implements `Table` which is widely used by many other graph types. In logs and during debugging, this could lead to a misunderstanding of what `SimpleTable` actually is.

Renamed for clarity.